### PR TITLE
Add read me

### PR DESCRIPTION
**Pull request by deborahvandervegt**
Closes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Windmill Academy Addon
+Addon to implement an academy module in an eportfolio
+
+## Installation
+Add the addon in your composer json:
+```
+{
+    "require": {
+        "connectholland/academy-addon": "dev-add-read-me@dev"
```
Comment (RonRademaker) [inline README.md]: You shouldn't suggest installing some dev branch in your readme. Instead, suggest dev-master until you got a release. With the first release you should update the readme to use a version.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    "repositories": [
+    	{
+    		"type": "vcs",
+    		"url": "https://github.com/ConnectHolland/Windmill-Academy-Addon"
+ 		}
+    ]
+}
+```
+Run composer install / update.
```
Comment (RonRademaker) [inline README.md]: Composer install won't work if you just added something.

File: README.md (hunk 1 above)
Comment (RonRademaker) [inline README.md]: Update: document the installation using composer commands:

``` bash
composer require connectholland/academy-addon dev-master
```

**Comment by RonRademaker** [comment]
You missed the activation of the addon in ```WMWindmill:registerAddons```

File: README.md (hunk 1 above)
Comment (deborahvandervegt) [inline README.md]: How do I make an specific version then?

**Comment by deborahvandervegt** [comment]
Added changed read me in new pull request.

**Pull request closed without merging**